pkgs/by-name/fo/fortune-kind/package.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, darwin
+, libiconv
+, makeWrapper
+, installShellFiles
+, fortuneAlias ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fortune-kind";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "cafkafk";
+    repo = pname;
```

```suggestion
    repo = "fortune-kind";
```
they're semantically different and should be treated as such